docs/en/cli.md

```diff
@@ -51,18 +51,18 @@ View tablet information (if the serverName and automatic local ip function are u
   6534708420092481536 172.17.0.14:9533 kTabletHealthy 14h
 ```
 
-#### addreplica
+### addreplica
 
-Add replica
+Add replicas
 
-Command format: `addreplica table_name pid_group endpoint`
+Command format: `addreplica the table name pid_group endpoint`
```

Comment:
this might be my mistake, your original word is correct. sorry. pls help change it back
the table name => table_name

Reply:
I'm careless, thank you.